bokehjs/src/lib/models/axes/axis.ts

```diff
@@ -567,8 +618,8 @@ export class Axis extends GuideRenderer {
       minor_tick_line_color: 'black',
 
       major_label_text_font_size: "11px",
-      major_label_text_align: "center",
-      major_label_text_baseline: "alphabetic",
+      major_label_text_align: "center",        // XXX: remove
+      major_label_text_baseline: "alphabetic", // XXX: remove
```

Comment:
Just noting that this issue could possiblyt be purposed for removing these in general https://github.com/bokeh/bokeh/issues/10868

Reply:
Here I wrote this should be removed, but really this should be generalized, especially for typesetting math and non-textual labels (i.e. baseline -> axis of alignment). Also alignment needs to be disambiguated (e.g. align left means to align lines in a multiline label or is it the attachment point of the bounding box of such label, or both).